### PR TITLE
Revert PublishCodeCoverageResults task docs to V1 definition

### LINKED ISSUE #11183: Missing parameters for PublishCodeCoverageResults
There are additional parameters missing from documentation.

Current list.

```yaml
- task: PublishCodeCoverageResults@1
    enabled: true
    inputs:
      codeCoverageTool: 
      summaryFileLocation: 
      failIfCoverageEmpty: false
      additionalCodeCoverageFiles: 
      pathToSources: 
      reportDirectory: 
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: c6ac4ff4-9839-6a6d-8d20-a25084d221ef
* Version Independent ID: bd09f139-8e69-8ea1-6671-b7bba8e17c23
* Content: [Publish Code Coverage Results task - Azure Pipelines](https://docs.microsoft.com/en-us/azure/devops/pipelines/tasks/test/publish-code-coverage-results?view=azure-devops)
* Content Source: [docs/pipelines/tasks/test/publish-code-coverage-results.md](https://github.com/MicrosoftDocs/azure-devops-docs/blob/master/docs/pipelines/tasks/test/publish-code-coverage-results.md)
* Product: **devops**
* Technology: **devops-cicd-tasks**
* GitHub Login: @shashban
* Microsoft Alias: **shashban**

### PR DESCRIPTION
**Pull request by berviantoleo**
I send this PR because I can't use the V2 tasks:

https://github.com/microsoft/azure-pipelines-tasks/issues/15160

Also, there is another person ask about this, hope it will fix #11183.

If the V2 task is available, I will send the PR to update the snippets to V2 instead.

Thanks.

### CODE REVIEW

**Comment by PRMerger10** [comment]
@berviantoleo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/pipelines/tasks/includes/yaml/PublishCodeCoverageResultsV1.md [hunk 1]
```diff
@@ -12,7 +12,10 @@ ms.technology: devops-cicd-tasks
 # Publish Cobertura or JaCoCo code coverage results from a build
 - task: PublishCodeCoverageResults@1
   inputs:
+    #codeCoverageTool: 'JaCoCo' # Options: cobertura, jaCoCo
```
Comment (DenisRumyantsev) [inline docs/pipelines/tasks/includes/yaml/PublishCodeCoverageResultsV1.md:15]:
```suggestion
    codeCoverageTool: 'JaCoCo' # Available options: 'JaCoCo', 'Cobertura'
```

Comment (berviantoleo) [reply docs/pipelines/tasks/includes/yaml/PublishCodeCoverageResultsV1.md]: Thanks!

**Comment by PRMerger10** [comment]
@berviantoleo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

**Review by anatolybolshakov — APPROVED** [review]

File: docs/pipelines/tasks/test/publish-code-coverage-results.md [hunk 2]
```diff
@@ -50,6 +50,9 @@ To publish code coverage results for JavaScript with Istanbul using YAML, see [J
...
 |`pathToSources` <br/>Path to Source files|(Optional) Path to source files is required when coverage XML reports do not contain absolute path to source files. <br/>For example, JaCoCo reports do not use absolute paths and when publishing JaCoCo coverage for Java apps, the pattern would be similar to `$(System.DefaultWorkingDirectory)/MyApp/src/main/java/`. <br/>This input is also needed if tests are run in a docker container. This input should point to absolute path to source files on the host. <br/>For example, `$(System.DefaultWorkingDirectory)/MyApp/`|
 |`failIfCoverageEmpty`<br/>Fail if code coverage results are missing|(Optional) Fail the task if code coverage did not produce any results to publish.|
+|`reportDirectory`<br/>Report directory|Path of the code coverage HTML report directory. The report directory is published for later viewing as an artifact of the build. The value may contain minimatch patterns. For example: `$(System.DefaultWorkingDirectory)/MyApp/**/site/cobertura`|
```
Comment (DenisRumyantsev) [inline docs/pipelines/tasks/test/publish-code-coverage-results.md:53]:
```suggestion
|`reportDirectory`<br/>Report directory|(Optional) Path of the code coverage HTML report directory. The report directory is published for later viewing as an artifact of the build. The value may contain minimatch patterns. For example: `$(System.DefaultWorkingDirectory)/MyApp/**/site/cobertura`|
```

File: docs/pipelines/tasks/test/publish-code-coverage-results.md [hunk 3]
```diff
@@ -50,6 +50,9 @@ To publish code coverage results for JavaScript with Istanbul using YAML, see [J
...
+|`additionalCodeCoverageFiles`<br/>Additional files|File path pattern specifying any additional code coverage files to be published as artifacts of the build. The value may contain minimatch patterns. For example: `$(System.DefaultWorkingDirectory)/**/*.exec`|
```
Comment (DenisRumyantsev) [inline docs/pipelines/tasks/test/publish-code-coverage-results.md:54]:
```suggestion
|`additionalCodeCoverageFiles`<br/>Additional files|(Optional) File path pattern specifying any additional code coverage files to be published as artifacts of the build. The value may contain minimatch patterns. For example: `$(System.DefaultWorkingDirectory)/**/*.exec`|
```

**Comment by PRMerger9** [comment]
@berviantoleo : Thanks for your contribution! The author(s) have been notified to review your proposed change.

File: docs/pipelines/tasks/test/publish-code-coverage-results.md (hunk 2 above)
Comment (berviantoleo) [reply docs/pipelines/tasks/test/publish-code-coverage-results.md]: Thanks!

File: docs/pipelines/tasks/test/publish-code-coverage-results.md (hunk 3 above)
Comment (berviantoleo) [reply docs/pipelines/tasks/test/publish-code-coverage-results.md]: Thanks!

**Review by DenisRumyantsev — APPROVED** [review]

**Comment by DenisRumyantsev** [comment]
I am closing this PR since it has been transferred to another repository to speed up its merging.

**Comment by berviantoleo** [comment]
May I know where?

**Comment by DenisRumyantsev** [comment]
@berviantoleo, this other repository has limited access. The PR was merged — thank you for provided changes! Let me know in [the related ticket](https://github.com/microsoft/azure-pipelines-tasks/issues/15160) if it can be closed or we need here anything else.

**Comment by berviantoleo** [comment]
Ok, thank you. I've updated at related issues too.

**Pull request closed without merging**